### ISSUE TITLE
do you have a plan to use getOptions instead of parseQuery?

### ISSUE THREAD

**Issue by bitcldr** [AngularClass/angular2-hmr-loader#7]
do you have a plan to use getOptions instead of parseQuery? see details here https://github.com/webpack/loader-utils/issues/56

**Comment by MattDiMu** [comment]
Also getting this deprecation warning:

```
(node:18544) DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.
```

(It should be enough to simply replace "parseQuery" with "getOption")

**Comment by PatrickJS** [comment]
can you make a PR?

**Comment by acostaf** [comment]
@gdi2290 @MattDiMu I've done the change but now I need permission to git push it to you, Can I've permission.

**Comment by DcsMarcRemolt** [comment]
@acostaf Giving write access on a project repo to anyone having a patch is a bad idea. That's what pull requests are used for. 

https://help.github.com/articles/about-pull-requests/

Please fork the main repo, commit and push the change, then issue a pull request.

**Comment by MattDiMu** [comment]
@gdi2290 Looks good - could you publish the new version to npm as well (3.0.3)?

**Comment by PatrickJS** [comment]
@MattDiMu thanks, it's published as 3.0.3

**Comment by MattDiMu** [comment]
@gdi2290 Unfortunately the published package (3.0.3) still uses parseQuery on index.js:10. Are you sure you published the correct version?

Steps to reproduce:
1. npm pack @angularclass/hmr-loader (downloads the tarball)
2. untar/unzip the tarball
3. look into index.js

**Comment by PatrickJS** [comment]
@MattDiMu alright it should be fixed as 
@angularclass/hmr-loader@3.0.4
